### DISCUSSION TITLE
How to multiple invoices in a single document?

### DISCUSSION THREAD

**Discussion by drumset** [QuestPDF/QuestPDF#634] [Q&A] [answered]
Great job, really. Testing the example invoice, I have seen that there is only one invoice , but how to do it for multiple invoices in a single document?
For example, a list of invoices from a client.
Thank you.

**Comment by drumset** [comment, marked as answer]
I answer myself. 
` Document.Merge(invoices).UseOriginalPageNumbers().GeneratePdf(filePath);
`

Thank you.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
